package/Dockerfile.shipyard-dapper-base

```diff
@@ -1,4 +1,4 @@
-FROM fedora:35
+FROM fedora:36
```

Comment:
(tl;dr this works both pre and post-release)
It seems like this tag is maintained at the latest build from the 36 branch, which is currently pre-release but will be updated to 36 once it's released, and will continue to be updated after release (based on 34/35 tags I see) 


https://quay.io/repository/fedora/fedora?tab=tags